package.json

```diff
@@ -64,14 +62,13 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-n": "^15.0.0",
     "eslint-plugin-perfectionist": "^1.4.0",
+    "eslint-plugin-preact": "^0.1.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-promise": "^6.0.0",
-    "eslint-plugin-react": "^7.32.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.4.1",
     "eslint-plugin-tailwindcss": "^3.13.0",
     "jsdom": "^22.1.0",
     "prettier": "^3.0.2",
+    "react-use-clipboard": "^1.0.9",
```

Comment:
* It's _main_ dependancy, not `devDependancy`
* Since it's react lib it should work through preact-compat layer
* Since it's react lib let's find a lib for preact or rewrite it from scratch (should not be hard).